src/dotnet/commands/dotnet-compile-native/ArgumentsParser.cs

```diff
@@ -1,154 +1,124 @@
 ﻿using System;
 using System.Collections.Generic;
-using System.CommandLine;
 using System.Linq;
+using Microsoft.DotNet.Cli.CommandLine;
+using Microsoft.DotNet.Cli.Utils;
 
```

Comment:
@Petermarcu @piotrpMSFT It would be ideal if we got either a `Microsoft.DotNet.Cli.CommandLine` for `CLI extensions` or `Microsoft.Extensions.CommandLine` from `CoreFX` for everyone to use, it would be perfect. `fsc`, `ilc`, `csc` tools/extensions all share this code and forking seems like going back in time when `System.CommandLine` did not exist.